tests/bitset_handler/test_bitset_pandas_dtype.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+import pytest
+from pandas.tests.extension.base import BaseDtypeTests
+
+from tlo.bitset_handler.bitset_extension import BitsetDtype
+
+
+class TestMyDtype(BaseDtypeTests):
```

Comment:
```suggestion
class TestBitsetDtype(BaseDtypeTests):
```

Presumably name was taken from pandas example code, but probably something more specific here would make sense!

Reply:
No idea what you mean, totally didn't need to borrow code from any examples here... :innocent: